.markdownlint.yaml

```diff
@@ -0,0 +1 @@
+MD034: false
```

Comment:
the one rule we still need to skip because we need bare URLs for github code snippets